src/api/database/partitions.rst

```diff
@@ -76,10 +75,10 @@ such that each partition can be queried meaningfully.
 ==================================
 
 .. http:post:: /{db}/_partition/{partition}/_find
-    :synopsis: Find documents inside a specific partition
-
-    Find documents inside a specific partition, see the main
-    documentation for :ref:`_find <api/db/_find>`.
+    :synopsis: Finds documents inside the specified partition
+    
+    Finds documents inside the specified partition. See the CouchDB
+    documentation for more information about :ref:`_find <api/db/_find>`.
```

Comment:
"See the CouchDB documentation" from within the CouchDB documentation seems possibly confusing to me. Maybe we can re-word this, to something like:

> For more information, see :ref:`_find <api/db/_find>`.

?